Jenkinsfile.treecompose

```diff
@@ -92,7 +94,9 @@ node(NODE) {
             version = utils.get_rev_version("${repo}", commit)
             currentBuild.description = "🆕 ${version} (commit ${commit})";
             if (previous_commit.length() > 0) {
-                sh "rpm-ostree --repo=${repo} db diff ${previous_commit} ${commit}"
+                sh "mkdir -p ${images}"
+                utils.rsync_dir_in(ARTIFACT_SERVER, KEY_FILE, images)
```

Comment:
If `images` is the best place for this file to live, should I create a `sync in` stage?

Reply:
You don't need to sync anything in for this operation.  Just making the directory and writing out the pkg diff should be sufficient.

Reply:
I noticed the `cloud` pipeline sync it here: https://github.com/openshift/os/blob/master/Jenkinsfile.cloud#L36. If it isn't sync, wouldn't files/directories get overwritten during treecompose's `rsync-out` stage?

Reply:
ignore ^, `archiveAritifacts` solves this

Reply:
As I understand it, in the `cloud` pipeline, that sync in is used so that the previous `commit.txt` artifact can be referenced in the `Check for changes` stage:

https://github.com/openshift/os/blob/master/Jenkinsfile.cloud#L60

Since writing out the pkg diff doesn't require the knowledge of any previous artifacts in that `images` directory, I believe the `sync-in` can be skipped.